src/etcpal/lock.dox

```diff
@@ -3,34 +3,83 @@ platform-specific, and some of them implement types and functions differently (i
 macros, etc.), so the documentation needs somewhere to pull a standard interface from. Voila!
 
 /*!
- * \defgroup etcpal_lock Locks and Synchronization (lock)
```

Comment:
Are the .dox files autogenerated for you, or did you manually edit it?

Reply:
This is the source of the documentation for the lock module. It's edited manually.